hack/quick_install.sh

```diff
@@ -122,7 +122,7 @@ kubectl apply -f https://github.com/kserve/kserve/releases/download/${KSERVE_VER
 
 # Install KServe built-in servingruntimes
 kubectl wait --for=condition=ready pod -l control-plane=kserve-controller-manager -n kserve --timeout=300s
-kubectl apply -f https://github.com/kserve/kserve/releases/download/${KSERVE_VERSION}/kserve-runtimes.yaml
+#kubectl apply -f https://github.com/kserve/kserve/releases/download/${KSERVE_VERSION}/kserve-runtimes.yaml
```

Comment:
It should work so please uncomment this line.

Reply:
Thanks. I made the change.